contracts/AelinUpFrontDeal.sol

```diff
@@ -72,19 +74,25 @@ contract AelinUpFrontDeal is AelinERC20, MinimalProxyFactory, IAelinUpFrontDeal
 
         uint8 purchaseTokenDecimals = IERC20Decimals(_dealData.purchaseToken).decimals();
         require(purchaseTokenDecimals <= DEAL_TOKEN_DECIMALS, "purchase token not compatible");
-
-        require(1825 days >= _dealConfig.vestingCliffPeriod, "max 5 year cliff");
-        require(1825 days >= _dealConfig.vestingPeriod, "max 5 year vesting");
-
         require(_dealConfig.underlyingDealTokenTotal > 0, "must have nonzero deal tokens");
-        require(_dealConfig.purchaseTokenPerDealToken > 0, "invalid deal price");
 
+        require(_dealConfig.vestingSchedule.length > 0, "must have vesting schedule");
+        require(_dealConfig.vestingSchedule.length < MAX_VESTING_SCHEDULES, "exceeds max amount of vesting schedules");
         uint8 underlyingTokenDecimals = IERC20Decimals(_dealData.underlyingDealToken).decimals();
+        uint256 highestPrice;
```

Comment:
shouldn't it be lowestPrice instead? We want  `maxRaise = minimum price * underlyingDealTokenTotal` as it will be the minimum raise amount